tests/unit/downloader/test_data_collector.py

```diff
@@ -0,0 +1,60 @@
+import json
+import unittest
+from unittest.mock import MagicMock
+
+from ingest.api.ingestapi import IngestApi
+from ingest.downloader.data_collector import DataCollector
+
+
+class MyTestCase(unittest.TestCase):
```

Comment:
I usually see the format `<ClassName>Test` for test class names.

Reply:
I changed it. Tx